web/src/engine/osk/src/input/gestures/browser/keytip.ts

```diff
@@ -5,7 +5,7 @@ import VisualKeyboard from '../../../visualKeyboard.js';
 import { GesturePreviewHost } from '../../../keyboard-layout/gesturePreviewHost.js';
 
 const CSS_PREFIX = 'kmw-';
-const DEFAULT_TIP_ORIENTATION = 'top';
+const DEFAULT_TIP_ORIENTATION: PhoneKeyTipOrientation = 'top';
 
 export type PhoneKeyTipOrientation = 'top' | 'bottom';
```

Comment:
This should be above line 8, even though it seems to build okay, just for clarity.